### PR TITLE
chore(benchpress): add ngRepeat animation benchmark

### PR DESCRIPTION
**Pull request by Narretz**

### CODE REVIEW

File: benchmarks/repeat-animate-bp/common.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+        function createList() {
+          for (i = 0; i < $scope.data.length; i++) {
+            row = document.createElement('div');
+            $element[0].appendChild(row);
+            for (j = 0; j < $scope.data[i].length; j++) {
+              cell = template.cloneNode(true);
+              row.appendChild(cell);
+              cell.childNodes[0].textContent = i;
+              cell.ng3992 = 'xxx';
```
Comment (gkalpak) [inline benchmarks/repeat-animate-bp/common.js]: I thought the magic number was `339`...

File: benchmarks/repeat-animate-bp/main.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<div ng-app="repeatAnimateBenchmark" ng-cloak>
+  <div ng-controller="DataController">
+    <div class="container-fluid">
+      <p>
+        Tests rendering of an ngRepeat with 1000 elements.<br>
```
Comment (gkalpak) [inline benchmarks/repeat-animate-bp/main.html]: _1000_  -->  _500_ (?)

Comment (Narretz) [inline benchmarks/repeat-animate-bp/main.html]: 500 actually :)

File: benchmarks/repeat-animate-bp/common.js (hunk 1 above)
Comment (Narretz) [inline benchmarks/repeat-animate-bp/common.js]: :+1:

**Pull request closed without merging**